src/main/kotlin/org/medaware/anterogradia/libs/Standard.kt

```diff
@@ -49,6 +59,22 @@ class Standard(val runtime: Runtime) {
     @VariadicFunction(identifier = "random")
     fun random(strings: Array<Node>): String = strings.random().evaluate(runtime)
 
+    @DiscreteFunction(identifier = "param", params = ["key"])
+    fun parameter(id: Node): String = runtime.parameters[id.evaluate(runtime)] ?: ""
+
+    @DiscreteFunction(identifier = "set", params = ["key", "value"])
+    fun set(key: Node, value: Node): String {
+        variableStore[key.evaluate(runtime)] = value.evaluate(runtime)
+        return ""
+    }
+
+    @DiscreteFunction(identifier = "get", params = ["key"])
+    fun set(key: Node): String = variableStore[key.evaluate(runtime)] ?: ""
```

Comment:
isn't this meant to be ```get(key: Node)``` ?

Reply:
White it _technically_ wouldn't matter, as the `LibraryManager`'s `invokeLibMethod` function only _really_ cares about the annotation when looking for functions (below an excerpt from said function) ...
```kotlin
val rawMethods =
            lib.declaredMethods.filter { it.getAnnotation(DiscreteFunction::class.java)?.identifier == name }
```
.. it definitely is very confusing to read and something that I have missed. Thanks for pointing it out! Resolved in 44317c1.